File: node-api/get-deploy/request.go
package main

import (
	"fmt"
	"github.com/casper-ecosystem/casper-golang-sdk/sdk"
)

func main() {

	url := "http://3.136.227.9:7777/rpc"
	deployHash := "6c4048f8ebd40a160e9df47e73680eda8ae8430309a9566655bb357a5967276b"

	client, err := sdk.NewRpcClient(url)
	if err != nil {
		fmt.Println(err)
		return
	}
	deploy, err := client.GetDeploy(deployHash)
	if err != nil {
		fmt.Println(err)
		return
	}

	fmt.Println(deploy)
}

File: node-api/get-deploy/request.js
const { CasperServiceByJsonRPC } = require("casper-js-sdk");

// A known deploy hash in base16.
const _DEPLOY_HASH = "6c4048f8ebd40a160e9df47e73680eda8ae8430309a9566655bb357a5967276b"

const client = new CasperServiceByJsonRPC("http://3.136.227.9:7777/rpc");

client.getDeployInfo(
  _DEPLOY_HASH
).then(result => {
  console.log(result);
});

